### DISCUSSION TITLE
Collapse rules for patterns

### DISCUSSION THREAD

**Discussion by swehba** [jglev/obsidian-apply-patterns-plugin#18] [Ideas]
The display the patterns and their rules can get very messy. To decrease the "noise", it would be nice to be able to collapse all the rules for a given pattern or all patterns. I have been using another plugin for regex rulesets, and I very much prefer Apply Patterns *except* for not being able to collapse the patterns.

**Comment by jglev** [comment]
@swehba, I take your point! I've just now released [`1.3.0`](https://github.com/publicus/obsidian-apply-patterns-plugin/releases/tag/1.3.0), which adds the ability to collapse individual patterns, and to cycle through collapsing and expanding all visible patterns at once.

**Reply by swehba** [reply]
Cool. Thanks.

**Reply by jglev** [reply]
You're welcome!
